lib/2wp-utils.js

```diff
@@ -242,28 +242,24 @@ const donateToBridge = async (rskTxHelper, btcTxHelper, donatingBtcAddressInform
 };
 
 /**
- * Disabling whitelisting
+ * Disable the whitelisting in the bridge and mine the required blocks specified in the blockDelay parameter if mineDelayBlocks is true
  * @param {RskTransactionHelper} rskTxHelper 
  * @param {BtcTransactionHelper} btcTxHelper 
+ * @param {number} blockDelay default 1
+ * @param {boolean} mineDelayBlocks default true
```

Comment:
What if you pass 0 when you don't want to mine any blocks?

Reply:
Then the `btcTxHelper` will complain about it. If you call `btcTxHelper.mine` with a 0, it will through an exception. Now that I think about it, It's better if these optional parameters are reversed. Updated!

Reply:
I mean using 0 as the argument to mine anything. If you don't pass anything, the value will be zero, so when you check it is 0, you don't mine anything. What do you think?